ui-ngx/src/app/core/http/entity.service.ts

```diff
@@ -416,7 +418,11 @@ export class EntityService {
         break;
       case EntityType.WIDGETS_BUNDLE:
         pageLink.sortOrder.property = 'title';
-        entitiesObservable = this.widgetService.getWidgetBundles(pageLink, false, config);
+        entitiesObservable = this.widgetService.getWidgetBundles(pageLink, true, false, config);
```

Comment:
This code breaks components (tb-entity or tb-entity-list) when used in the tenant.